payments-ui-core/res/values/strings.xml

```diff
@@ -77,4 +77,8 @@
   <string name="stripe_setup_button_label">Set up</string>
   <!-- Label for UPI ID number field on form -->
   <string name="stripe_upi_id_label">UPI ID</string>
+  <!-- Label for Konbini Payment Method -->
+  <string name="stripe_paymentsheet_payment_method_konbini">Konbini</string>
```

Comment:
Should this be in the no-translate file?

Reply:
This is translated differently in Japanese.